installer

```diff
@@ -0,0 +1,131 @@
+#!/usr/local/cpython-3.4/bin/python3
+
+"""
+Copy modules from here to --prefix.
+
+Note that we do not scan for directories; instead we have a hardcoded list in all_directories.
+"""
+
+import os
+import shutil
+import sys
+
+
+prefix = os.path.expanduser('~/.micropython/lib')
+directories = []
+all_directories = ['micropython', 'python-ecosys', 'python-stdlib', 'tools', 'unix-ffi']
```

Comment:
tools shouldn't be copied probably, since it copies it codeformat.py which only works for this repository

Reply:
It's gone now.